.dotnet/Azure.AI.OpenAI.Staging/Custom/AzureOpenAIClient.cs

```diff
@@ -0,0 +1,281 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.OpenAI.Staging.Audio;
+using Azure.AI.OpenAI.Staging.Chat;
+using Azure.AI.OpenAI.Staging.Embeddings;
+using Azure.AI.OpenAI.Staging.FineTuning;
+using Azure.AI.OpenAI.Staging.Images;
+using Azure.Core;
+using OpenAI;
+using OpenAI.Assistants;
+using OpenAI.Audio;
+using OpenAI.Chat;
+using OpenAI.Embeddings;
+using OpenAI.Files;
+using OpenAI.FineTuning;
+using OpenAI.Images;
+using OpenAI.ModelManagement;
+using OpenAI.Moderations;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.AI.OpenAI.Staging;
+
+/// <summary>
+/// The top-level client for the Azure OpenAI service.
+/// </summary>
+/// <remarks>
+/// For scenario-specific operations, create a corresponding client using the matching method on this type, e.g. <see cref="GetChatClient(string)"/>.
+/// </remarks>
+public partial class AzureOpenAIClient : OpenAIClient
+{
+    private const string azureOpenAIEndpointEnvironmentVariable = "AZURE_OPENAI_ENDPOINT";
+    private const string azureOpenAIApiKeyEnvironmentVariable = "AZURE_OPENAI_API_KEY";
+
+    private readonly AzureOpenAIClientOptions _options;
+
+    internal static PipelineMessageClassifier PipelineMessageClassifier200 => _pipelineMessageClassifier200.Value;
```

Comment:
\<incredibly nit-picky and stylistic\> in our generated code we put these at the bottom of the file, consider following that pattern, but I don't feel strongly about that at all </>

Reply:
I do want to follow consistent style! Does that apply to all private/internal non-instance fields? I'll move them.